src/components/__tests__/AutocompleteInput.spec.ts

```diff
@@ -227,13 +227,13 @@ describe('AutocompleteInput.vue', () => {
       document.body.appendChild(elem)
     }
     wrapper = mount(AutocompleteInput, {
-      propsData: {
+      props: {
         suggestions: suggestions,
         autoFocus: false,
         clearInputAfterSelection: false,
       },
       attachTo: elem,
-    })
+    }) as VueWrapper<any>
```

Comment:
Maybe

```suggestion
    }) as VueWrapper<AutocompleteInput>
```

orMaybe

```suggestion
    }) as VueWrapper
```